assets/docs/neso613/ASR_TFLite/1.md

```diff
@@ -0,0 +1,10 @@
+# Placeholder neso613/ASR_TFLite/1
+[Conformer Transducer](https://arxiv.org/abs/2005.08100) using [LibriSpeech](http://www.openslr.org/12) dataset.
+
+<!-- dataset: LibriSpeech -->
+<!-- task: automatic speech recognition -->
```

Comment:
The task must start with any of 'image-', 'text', 'audio-', or 'video-'. For example,  `audio-transcription` might make sense for this model.

Reply:
ok, Thanks for suggestion. I'll update.